src/20_Components/amp-bind.html

```diff
@@ -0,0 +1,106 @@
+<!---{
+    "experiments": ["amp-bind"],
```

Comment:
Actually changed this to lowercase recently to be consistent with other experiments.